gateway/src/main/java/io/camunda/zeebe/gateway/impl/probes/health/HealthZeebeClientProperties.java

```diff
@@ -0,0 +1,236 @@
+      }
+
+      public void setClientId(final String clientId) {
+        this.clientId = clientId;
+      }
+
+      public String getClientSecret() {
+        return clientSecret;
+      }
+
+      public void setClientSecret(final String clientSecret) {
+        this.clientSecret = clientSecret;
+      }
+
+      public String getAudience() {
+        return audience;
+      }
+
+      public void setAudience(final String audience) {
+        this.audience = audience;
+      }
+
+      public URL getAuthorizationServer() {
+        return authorizationServer;
+      }
+
+      public void setAuthorizationServer(final URL authorizationServer) {
+        this.authorizationServer = authorizationServer;
+      }
+
+      public String getCredentialsCachePath() {
+        return credentialsCachePath;
+      }
+
+      public void setCredentialsCachePath(final String credentialsCachePath) {
+        this.credentialsCachePath = credentialsCachePath;
+      }
+
+      public File getCredentialsCache() {
+        return credentialsCache;
+      }
+
+      public void setCredentialsCache(final File credentialsCache) {
+        this.credentialsCache = credentialsCache;
+      }
+
+      public Duration getConnectTimeout() {
+        return connectTimeout;
+      }
+
+      public void setConnectTimeout(final Duration connectTimeout) {
+        this.connectTimeout = connectTimeout;
+      }
+
+      public Duration getReadTimeout() {
+        return readTimeout;
+      }
+
+      public void setReadTimeout(final Duration readTimeout) {
+        this.readTimeout = readTimeout;
+      }
+
+      @Override
+      public int hashCode() {
+        return Objects.hash(
+            clientId,
+            clientSecret,
+            audience,
+            authorizationServer,
+            credentialsCachePath,
+            credentialsCache,
+            connectTimeout,
+            readTimeout);
+      }
+
+      @Override
+      public boolean equals(final Object o) {
+        if (this == o) {
+          return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+          return false;
+        }
+        final OAuthSecurityProperties that = (OAuthSecurityProperties) o;
+        return Objects.equals(clientId, that.clientId)
+            && Objects.equals(clientSecret, that.clientSecret)
+            && Objects.equals(audience, that.audience)
+            && Objects.equals(authorizationServer, that.authorizationServer)
+            && Objects.equals(credentialsCachePath, that.credentialsCachePath)
+            && Objects.equals(credentialsCache, that.credentialsCache)
+            && Objects.equals(connectTimeout, that.connectTimeout)
+            && Objects.equals(readTimeout, that.readTimeout);
+      }
+
+      @Override
+      public String toString() {
+        return "OAuthSecurityProperties{"
+            + "clientId='"
+            + clientId
+            + '\''
+            + ", clientSecret='"
+            + clientSecret
```

Comment:
❌ Let's remove the client secret so we don't leak this.

Reply:
Ooops, missed that 😅 Thanks, I fixed this :)